# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../cinelerra/stray-backslash.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- cinelerra-2.0/global_config_orig	2006-04-11 18:51:59.301531500 +0200
+++ cinelerra-2.0/global_config	2006-04-11 18:52:17.178648750 +0200
@@ -83,7 +83,7 @@
 CXXREPOSITORY = cxx_repository/*.o
 endif
 
-$(shell sh -c 'if ! test -d $(OBJDIR)\; then mkdir $(OBJDIR)\; fi' )
+$(shell sh -c 'if ! test -d $(OBJDIR) ; then mkdir $(OBJDIR) ; fi' )
 
 ########################## LINKER
 
--- cinelerra-2.0/cinelerra/Makefile_orig	2006-04-11 18:52:52.396849750 +0200
+++ cinelerra-2.0/cinelerra/Makefile	2006-04-11 18:53:04.309594250 +0200
@@ -436,7 +436,7 @@
 
 
 
-$(shell echo "\#define BUILDDATE \"`date`\"" > builddate.h )
+$(shell echo "#define BUILDDATE \"`date`\"" > builddate.h )
 $(shell echo $(CFLAGS) > $(OBJDIR)/c_flags)
 $(shell echo $(LDFLAGS1) $(OBJS) $(DCRAW) $(FILEEXR) $(AVIOBJS) $(CXXREPOSITORY) $(STATICLIBS) $(LIBS) $(LDFLAGS2) > $(OBJDIR)/objs)
 
--- cinelerra-2.0/quicktime/Makefile_orig	2006-04-11 19:05:01.586421250 +0200
+++ cinelerra-2.0/quicktime/Makefile	2006-04-11 19:05:24.151831500 +0200
@@ -240,7 +240,7 @@
 LIBS := $(OUTPUT) $(MPEG3_DIR)/$(OBJDIR)/libmpeg3.a -lpng -lz -lpthread -ldl -lm
 
 
-$(shell if ! test -d $(OBJDIR) \; then mkdir -p $(DIRS) \; fi )
+$(shell if ! test -d $(OBJDIR) ; then mkdir -p $(DIRS) ; fi )
 $(shell echo  $(CFLAGS) > $(OBJDIR)/c_flags)
 $(shell echo  $(LAME_CFLAGS) > $(OBJDIR)/lame_flags)
 $(shell echo  $(OBJS) $(JPEG_LIB)/*.o $(DV_LIB)/*.o $(FAAD_LIB)/*.o $(FAAC_LIB)/*.o $(FFMPEG_LIB)/*.o $(X264_LIB)/*.o $(OGG_LIB)/*.o $(VORBIS_LIB)/*.o $(LAME_OBJS) > $(OBJDIR)/objs)
